dockers/base-cuda/Dockerfile

```diff
@@ -50,8 +54,10 @@ RUN \
         libopenmpi-dev \
         openmpi-bin \
         ssh \
-    && \
+        libnccl2=$TO_INSTALL_NCCL \
+        libnccl-dev=$TO_INSTALL_NCCL
 
+RUN \
```

Comment:
why have this separately? This would just produce more layers -> more overhead

Reply:
Docker was complaining to me that in future releases, empty continuation lines are going to be errors in dockerfiles. Yes, it is a bit of overhead, but not too much

Reply:
I would remove the empty lines rather than splitting it into multiple `RUN`s 👀

Reply:
Alright, removed empty lines